src/main/java/com/jmb/springfactory/controller/api/ProductionScheduleController.java

```diff
@@ -33,19 +36,23 @@
     }
 
     @GetMapping
-    public List<ProductionScheduleDto> findAll(@RequestParam(value = "name", required = false) String name,
+    public PageDto<ProductionScheduleDto> findAll(@RequestParam(value = "name", required = false) String name,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)